tests/functional/sync_v3_sign_up.js

```diff
@@ -56,9 +51,12 @@ define([
       return this.remote.then(clearBrowserState());
     },
 
-    'Fx <= 56, verify at CWTS': function () {
+    'Fx <= 57, verify at CWTS': function () {
       return this.remote
-        .then(openPage(SIGNUP_FX_55_PAGE_URL, selectors.SIGNUP.HEADER, {
+        .then(openPage(SIGNUP_PAGE_URL, selectors.SIGNUP.HEADER, {
+          query: {
+            forceUA: uaStrings['desktop_firefox_57']
```

Comment:
Definitely an improvement over the duplicate URLs that were used before, nice one!